### PR TITLE
Check lrecl

### PR DESCRIPTION
**Pull request by tiantn**
## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ x] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]

File: packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts [hunk 1]
```diff
@@ -86,7 +89,9 @@ describe("FtpMvsApi", () => {
     });
 
     it("should upload content to dataset.", async () => {
-        const localFile = "/tmp/testds1.txt";
+        const localFile = tmp.tmpNameSync({ tmpdir: "/tmp" });
```
Comment (std4lqi) [inline packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts:92]: How about other cases using `/tmp/testds1.txt`?

File: packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts [hunk 2]
```diff
@@ -16,11 +16,14 @@
 import { FtpMvsApi } from "../../../src/ZoweExplorerFtpMvsApi";
 import { DataSetUtils } from "@zowe/zos-ftp-for-zowe-cli";
 import TestUtils from "../utils/TestUtils";
+import * as tmp from "tmp";
 
 // two methods to mock modules: create a __mocks__ file for zowe-explorer-api.ts and direct mock for extension.ts
 jest.mock("../../../__mocks__/@zowe/zowe-explorer-api.ts");
 jest.mock("../../../src/extension.ts");
 
+//jest.mock("../../../src/ZoweExplorerFtpMvsApi.ts");
```
Comment (std4lqi) [inline packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts:25]: Shall we delete this line?

File: packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpMvsApi.ts [hunk 3]
```diff
@@ -141,6 +150,22 @@ export class FtpMvsApi extends AbstractFtpApi implements ZoweExplorerApi.IMvs {
                     throw new Error();
                 }
             }
+            const lrecl: number = dsAtrribute.apiResponse.items[0].lrecl;
+            const data = fs.readFileSync(inputFilePath, "UTF-8");
+            const lines = data.split(/\r?\n/);
+            const foundIndex = lines.findIndex((line) => line.length > lrecl);
+            if (foundIndex !== -1) {
+                const message1 = `zftp Warning: At least one line, like line ${foundIndex + 1},
```
Comment (std4lqi) [inline packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpMvsApi.ts:158]: Extra space at the end seems unnecessary?

File: packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts (hunk 1 above)
Comment (tiantn) [reply packages/zowe-explorer-ftp-extension/__tests__/__unit__/Mvs/ZoweExplorerFtpMvsApi.unit.test.ts:90]: Thanks @std4lqi , I updated it.

**Review by std4lqi — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Pull request merged**